sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -47,7 +47,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>2.0.3</version> <!-- {x-version-update;com.azure:azure-core-amqp;dependency} -->
+      <version>2.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;dependency} -->
```

Comment:
This should already be in master.

Reply:
This branch was created based off the march release branch commit point. I still am trying to release before April release.